boards/arm/stm32g081b_eval/stm32g081b_eval.dts

```diff
@@ -105,6 +105,22 @@
 	status = "okay";
 };
 
+&tim15_ch1_pc1 {
+	pinmux = <STM32_PINMUX('C', 1, AF2)>;
```

Comment:
Remove this line, there is no change vs existing prop in node definition.

Reply:
Done